script.js

```diff
@@ -1,4 +1,345 @@
+  // Internal - console log
+  console.log(playerOneRollOne, playerOneRollTwo);
+  console.log(playerTwoRollOne, playerTwoRollTwo);
+};
+
+// LEADERBOARD HELPER FUNCTION - SUM OF ELEMENTS IN ARRAYS
+var arraySum = function (array) {
+  var sum = 0;
+  var arrayLength = array.length;
+  for (var i = 0; i < arrayLength; i += 1) {
+    sum += array[i];
+  }
+  return sum;
+};
+
+var displayLeaderboard = function () {
+  // First scenario - Player 1 wins
+  if (arraySum(playerOneNumberArray) > arraySum(playerTwoNumberArray)) {
+    return `<b>&#x1F31F LEADERBOARD &#x1F31F</b> </br>
+    </br>
+    Player 1: ${arraySum(playerOneNumberArray)} </br>
+    </br>
+    Player 2: ${arraySum(playerTwoNumberArray)} </br>
+    </br>
+    <b> PLAYER 1 IS LEADING!</b> </br>
+    </br>
+    </br>
+    Thank you for playing! &#x1F604 </br>
+    </br>
+    Click submit for another round! </br>
+    </br>`;
+  }
+
+  // Second scenario - Player 2 wins
+  else if (arraySum(playerOneNumberArray) < arraySum(playerTwoNumberArray)) {
+    return `<b>&#x1F31F LEADERBOARD &#x1F31F</b> </br>
+        </br>
+        Player 1: ${arraySum(playerOneNumberArray)} </br>
+        </br>
+        Player 2: ${arraySum(playerTwoNumberArray)} </br>
+        </br>
+        <b> PLAYER 2 IS LEADING! </b> </br>
+        </br>
+        </br>
+        Thank you for playing! &#x1F604 </br>
+        </br>
+        Click submit for another round! </br>
+        </br>`;
+  }
+
+  // Third scenario - Draw
+  else {
+    return `<b>&#x1F31F LEADERBOARD &#x1F31F</b> </br>
+        </br>
+        Player 1: ${arraySum(playerOneNumberArray)} </br>
+        </br>
+        Player 2: ${arraySum(playerTwoNumberArray)} </br>
+        </br>
+        <b> IT'S A DRAW! </b> </br>
+        </br>
+        </br>
+        Thank you for playing! &#x1F604 </br>
+        </br>
+        Click submit for another round! </br>
+        </br>`;
+  }
+};
+
+// WINNING CALCULATION HELPER FUNCTION
+// If to consider 2 scenarios - either Player 1 or Player 2 wins
+var winningCalculation = function () {
+  // First scenario - Player 1 wins
+  if (selectedNumbers[0] > selectedNumbers[1]) {
+    return `<b>&#x1F3B2 PLAYER 1 WINS &#x1F3B2</b> </br>
+        </br>
+        Player 1: ${selectedNumbers[0]} </br>
+        </br>
+        Player 2: ${selectedNumbers[1]} </br>
+        </br>
+        `;
+  }
+
+  // Second scenario - Player 2 wins
+  else if (selectedNumbers[0] < selectedNumbers[1]) {
+    return `<b>&#x1F3B2 PLAYER 2 WINS &#x1F3B2</b> </br>
+              </br>
+              Player 1: ${selectedNumbers[0]} </br>
+              </br>
+              Player 2: ${selectedNumbers[1]} </br>
+              </br>
+             `;
+  }
+
+  // Third scenario - Draw
+  else {
+    return `<b>&#x1F3B2 IT'S A DRAW &#x1F3B2</b> </br>
+              </br>
+              Player 1: ${selectedNumbers[0]} </br>
+              </br>
+              Player 2: ${selectedNumbers[1]} </br>
+              </br>
+              `;
+  }
+};
+
```

Comment:
your code from line 105- 135 is quite repeatable, try this instead: 

`
var winningCalculation = function () {

   var message = ""
  // First scenario - Player 1 wins
  if (selectedNumbers[0] > selectedNumbers[1]) {
     message = "Player 1 Wins!"
  }

  // Second scenario - Player 2 wins
  else if (selectedNumbers[0] < selectedNumbers[1]) {
      message = "Player 2 Wins!"
  }

  // Third scenario - Draw
  else {
      message = "It's a Draw"
  }

    return `<b>&#x1F3B2 ${message} &#x1F3B2</b> </br>
              </br>
              Player 1: ${selectedNumbers[0]} </br>
              </br>
              Player 2: ${selectedNumbers[1]} </br>
              </br>
              `;
};
`

Reply:
doing it in such a way keeps it shorter and cleaner! you will also be repeating much less code!